deploy/docker/docker-entrypoint.sh

```diff
@@ -93,99 +92,114 @@ fi
 
 # Fix issue #42 - export env EMQX_DASHBOARD__DEFAULT_USER__PASSWORD to configure
 # 'dashboard.default_user.password' in etc/plugins/emqx_dashboard.conf
-if [[ ! -z "$EMQX_ADMIN_PASSWORD" ]]; then
+if [[ -n "$EMQX_ADMIN_PASSWORD" ]]; then
     export EMQX_DASHBOARD__DEFAULT_USER__PASSWORD=$EMQX_ADMIN_PASSWORD
 fi
 
-# echo value of $VAR_FULL_NAME hiding secrets if any
-echo_value () {
+# echo value of $VAR hiding secrets if any
+# SYNOPSIS
+#     echo_value KEY VALUE
+echo_value() {
```

Comment:
Prevent use global variable across functions.
Explicit pass arguments to `echo_value`.